packages/react-debug-tools/src/ReactDebugHooks.js

```diff
@@ -797,8 +796,7 @@ function setupContexts(contextMap: Map<ReactContext<any>, any>, fiber: Fiber) {
   let current: null | Fiber = fiber;
   while (current) {
     if (current.tag === ContextProvider) {
-      const providerType: ReactProviderType<any> = current.type;
-      const context: ReactContext<any> = providerType._context;
+      const context: ReactContext<any> = current.type;
```

Comment:
Would this work with previous versions of React, debugged by fresh build (with these changes) of React DevTools?

Should be keep both variants (`current.type?.context || current.type` for backwards-compatibility?

Reply:
Hmm yeah, I forgot this is supposed to work for multiple versions.